src/main/kotlin/interactor/GetCitiesHasTheCheapestBananaPricesInteractor.kt

```diff
@@ -0,0 +1,22 @@
+package interactor
+
+import model.CityEntity
+
+
+class GetCitiesHasTheCheapestBananaPricesInteractor ( private val dataSource: CostOfLivingDataSource ){
+
+
+    fun execute(vararg cities: CityEntity): List<String> {
```

Comment:
the parameter 'cities' is never used. u should be using it inside your function

Reply:
i made some changes